packages/portfolio-deploy/src/portfolio-start.core.js

```diff
@@ -84,6 +84,7 @@ export const makePrivateArgs = async (
   /** @type {AxelarId} */
   const axelarIds = {
     Avalanche: axelarConfig.Avalanche.axelarId,
+    Ethereum: axelarConfig.Ethereum.axelarId,
```

Comment:
tricky... we probably need to maintain this for multichain-testing and such, but this core-eval already shipped to mainnet and we don't expect to run it again, so we have to remember that the code here is not the same as the code on mainnet.